modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -87,6 +87,22 @@ Current effective quota for the quota balancer, in bytes per second.
 
 ---
 
+=== vectorized_kafka_quotas_client_quota_throttle_time
```

Comment:
@pgellert Is this distinct from `vectorized_kafka_quotas_throttle_time` and can we provide a tip or further description to help distinguish between the two?

Reply:
Yeah, the naming is not amazing here. They are distinct.
* `vectorized_kafka_quotas_throttle_time` tracks throttle time based on [Broker-wide throughput limits](https://docs.redpanda.com/current/manage/cluster-maintenance/manage-throughput/#broker-wide-throughput-limits)
* `vectorized_kafka_quotas_client_quota_throttle_time` tracks throttle time based on [Client throughput limits](https://docs.redpanda.com/current/manage/cluster-maintenance/manage-throughput/#client-throughput-limits)